content/source/docs/cloud/workspaces/run-tasks.html.md

```diff
@@ -5,93 +5,91 @@ page_title: "Run Tasks - Workspaces - Terraform Cloud and Terraform Enterprise"
 
 # Run Tasks
 
--> **Note:** As of September 2021, Run Tasks are available only as a beta feature, are subject to change, and not all customers will see this functionality in their Terraform Cloud organization.
+-> **Note:** As of September 2021, Run Tasks are a private beta feature, subject to change, and not all customers will see this functionality in their Terraform Cloud organization.
 
 
-Run Tasks allow Terraform Cloud to execute tasks in external systems at specific points in the Terraform Cloud run lifecycle. The beta release of this feature allows users to add and execute these tasks during the new pre-apply stage which exists in between the plan and apply stages. Tasks are executed by sending an API payload to the external system. This payload contains a collection of run-related information and a callback URL which the external system can use to send updates back to Terraform Cloud.
+Run Tasks allow you to directly integrate third-party tools and services at certain stages in the Terraform Cloud run lifecycle. When executed, a run task will send an API payload to the external service. This payload contains a collection of run-related information, including a callback URL which the service uses to send run task data back to Terraform Cloud.
 
-The external system can then use this run information and respond back to Terraform Cloud with a passed or failed status. Terraform Cloud uses this status response to determine if a run should proceed, based on the task's enforcement settings within a workspace.
+The external service will use this run information to respond back to Terraform Cloud with a passed or failed status. Terraform Cloud uses this status response to determine if a run should proceed, based on the run task's enforcement settings that have been configured for the workspace.
 
 -> **API:** See the [Run Tasks APIs](../api/run-tasks.html).
```

Comment:
Avoiding adding another note - we want to use notes as sparingly as possible. Also, we want to be inclusive and not use "see". So, I replaced it  - lmk what you think! 

```suggestion
You can manage run tasks through the UI as detailed below or through the [Run Tasks APIs](../api/run-tasks.html).
```